### DISCUSSION TITLE
Improve mobile UX for "robot controls" popup

### DISCUSSION THREAD

**Discussion by slovdahl** [Hypfer/Valetudo#1466] [Archive]
With a big mobile screen, reaching the button for closing the "robot controls" popup is quite hard without changing the grip of the mobile device. Would it make sense to keep the "Valetudo" bar with the open/close button in the same place all the time? I guess it depends on the vertical screen size, but at least on my OnePlus 8 Pro it wouldn't make much difference for the placement of the controls, give or take a few pixels it can be placed at the same place.

That is, instead of the current situation:

![valetudo-1](https://user-images.githubusercontent.com/1417619/162634270-2b26fb15-285d-47f2-8ef9-f1fcf1bbf406.png)

Something like this:

![valetudo-2](https://user-images.githubusercontent.com/1417619/162634273-f94cb29b-5ff2-47ec-ab3c-79c3d1978987.png)

**Comment by Hypfer** [comment]
That's a very good idea!
I like it

**Reply by Hypfer** [reply]
Done

f7f3f0db6e155aff25344d293e9822b589e43d9a
